build/README.md

```diff
@@ -1,113 +1,110 @@
-*   kubectl
-    *   Required if deploying with Kubernetes.
-*   docker
-    *   Required if building new images of the DSS.
-*   Cockroachdb
-    *   Required to generate new CockroachDB certificates.
-*   Google Cloud SDK (if deploying on GCP)
-    * Required if deploying to Google Cloud.
-*   Golang.
-    *   Required if developing the DSS codebase.
-*   Optional - [Jsonnet](https://github.com/google/jsonnet)
-    * Recommended if editing the jsonnet templates.
-
+- Install the custom kubejsonnet binary for better config management. Run:
+  `go install ./cmds/kubejsonnet` from the root directory of this project.
+- The [kubecfg client](https://github.com/bitnami/kubecfg#install)
+  - Required if deploying using the defined Kubernetes templates.
+- kubectl
+  - Required if deploying with Kubernetes.
+- docker
+  - Required if building new images of the DSS.
+- Cockroachdb
+  - Required to generate new CockroachDB certificates.
+- Google Cloud SDK (if deploying on GCP)
+  - Required if deploying to Google Cloud.
+- Golang.
+  - Required if developing the DSS codebase.
+- Optional - [Jsonnet](https://github.com/google/jsonnet)
+  - Recommended if editing the jsonnet templates.
 
 ## Docker images
 
-
-The grpc-backend and http-gateway are the 2 main binaries for processing DSS 
+The grpc-backend and http-gateway are the 2 main binaries for processing DSS
 requests. These are both built and pushed to a docker registry of your choice.
 You can easily find out how to push to a docker registry through a quick search.
 All major cloud providers have a docker registry service, or you can set up your
 own.
 
-
 ## Deploying the DSS on Kubernetes
 
-
 Note: All DSS instances in the same cluster must point their ntpd at the
 same NTP Servers. [CockroachDB recommends](https://www.cockroachlabs.com/docs/stable/recommended-production-settings.html#considerations)
 using [Google's Public NTP](https://developers.google.com/time/) when
 running in a multi-cloud environment.
 
-This secion discusses deploying a Kubernetes service, although you can deploy 
+This secion discusses deploying a Kubernetes service, although you can deploy
 a DSS instance however you like as long as it meets the CockroachDB requirements
 above. You can do this on any supported
 [cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Consult the Kubernetes documentation for
+or even on your own infrastructure. Consult the Kubernetes documentation for
 which ever provider you choose.
 
 1.  Create a new Kubernetes cluster as mentioned above. We recommend a new
     cluster for each DSS instance.
 
+1.  Create a namespace for your Kubernetes cluster in which the DSS services will reside. This can be achieved with `kubectl create ns <NAMESPACE>`. Alternatively, you may want deploy in a namespace that already exists, so keep track of the namespace for later steps.
```

Comment:
Let's make a suggestion for the value of NAMESPACE.  Like `your-organization` or `dss-prod` or whatever an example of the appropriate class of names is.

Reply:
I don' think the namespace matters TBH.  They might want to deploy to a pre-existing namespace that does not follow any sort of convention, and it would work perfectly fine.

Reply:
That's fine, but let's tell the user that so they don't spend unnecessary mental energy trying to figure out whether/when it matters and/or what it should be.  How about adding at the end:

The namespace [affects DNS addresses](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#namespaces-and-dns), but the value chosen is not particularly important for these instructions.  [One convention](https://kubernetes.io/docs/tasks/administer-cluster/namespaces/#subdividing-your-cluster-using-kubernetes-namespaces) might be to choose a namespace of `production` to differentiate from `development`.